### ISSUE TITLE
Cant Boot getting Apple USB erorrs

### ISSUE THREAD

**Issue by adnanmur** [sash11/ASRock-B550-Phantom-Gaming-ITX-Ryzen-5900X#2]
Hi mate,

I took your EFI as i have the same board, unfrotunately i cant boot, and on verbose screen AppleUSB controller is getting closed and that error is in a loop. 
I have an efi that i can boot normally, but i have usb isssues and cant get around them so i though i will try yours but unfortunatelly i cant get pass that screen even if i remove your ssdt and usb kext. So it must be something in the config or bios settings.

Please if you have time to share your bios settings or any tips you have.

Thanks

Adnan

**Comment by sash11** [comment]
Hello. Sorry to hear about your troubles. Did you try to replace the USB map once you have a successful boot with my custom USB kext? Did that work for you?
I will take screen shots of the bios settings for you a bit later and post on the main page.

**Comment by adnanmur** [comment]
yes i tried to put your kext into a working efi, but no fix, i still getting constant usb disconects and only two usb ports that are working fine are keyboard and mouse. 

So this Hack is almost working except USB. 

The thing i noticed when i check in Hackintool, one usb controller is XHCI and the other is AppleSomething with xxxxx instead of the controller name. Can you also check please what controllers do you have? 
I think that you got some fix for this controller insider your config.plist and it is not applying to my motherboard. Ive spent countless hours troubleshooting this, and just booted windows besides macOS so i can chech the usb controller names and try to fix it from there.

There is no xhci handoff setting on this board and i think that could be a culprit.

**Comment by sash11** [comment]
I did not do much with USB as far as I remember. Initial install was done on Big Sur 11.2.3 which still supported the USB quirk. Then I did my usb map and upgraded the OS. I have to stay on Big Sur though. For some reason Intel lan boot argument does not work on Monterey - there is no uplink when I connect USB cable.

**Comment by sash11** [comment]
I added the bios settings in the Docs folder as a zip archive

**Comment by adnanmur** [comment]
Great, thanks alot for your effort, i will give it a few shots tommorow and see what comes up. If you have hackintool can you screenshot the usb section as well so i can see the controllers.

THanks a million

**Comment by sash11** [comment]
Uploaded

**Comment by sash11** [comment]
Not sure, but could be that HS10 is causing the boot loop for you. It prevented my system from proper reboot/shutdown until I disabled it from the custom USB map.

**Comment by adnanmur** [comment]
i will capture the photo of the error tommorow, It is something realted to AppleUSBSMC failing to hold the usb in ON state. I dont have access to to the machine now but this is what i get, found this online https://i.imgur.com/iWW9F1b.png

**Comment by adnanmur** [comment]
Hi mate, manage to make the hack work by creating a usb map from windows using usb toolbox. So far so good everything is working great, have not tested the ethernet network and bluetooth is not working and its better to use a dongle anyways. Thank you for your suppot.

**Comment by adnanmur** [comment]
To fix your Lan use LucyRTL8125Ethernet.kext Vers.(1.1.0d8)
